src/components/NavBar.tsx

```diff
@@ -17,9 +17,26 @@ function NavBar() {
       (item === 'Contact Us' && pageName === 'Contact')
     ) {
       return (
+        <NavContainer>
+          <NavItem
+            text={item}
+            isActive
+            onClick={() => {
+              if (item === 'Home') {
+                navigate('/')
+              } else {
+                navigate(`/${item.match('^[a-zA-Z-]*')}`)
+              }
+            }}
+          />
+          <BufferContainer />
+        </NavContainer>
+      )
+    }
```

Comment:
since the difference between the 2 return elements is only the isActive, you could just have one return version with the param isActive={<if condition>}

Reply:
fixed in 026258d